cloudflare.go

```diff
@@ -265,6 +267,17 @@ func (api *API) makeRequestWithAuthTypeAndHeaders(ctx context.Context, method, u
 				api.logger.Printf("Error performing request: %s %s : %s \n", method, uri, respErr.Error())
 			}
 			continue
+		}
+
+		// Successful request, let's read the body and return the first part
```

Comment:
Need to fix this comment, as the code is returning the _last_ part if there are multiple parts.